Integrations/src/main/java/io/deephaven/integrations/learn/Computer.java

```diff
@@ -87,7 +94,9 @@ public FutureOffset compute(long k) {
         current.getIndexSet().add(k);
         offset += 1;
 
-        return new FutureOffset(current, offset);
+        foSet.add(new FutureOffset(current, offset));
+
+        return foSet.get(offset);
```

Comment:
```suggestion
        final FutureOffset fo = new FutureOffset(current, offset);
        foSet.add(fo);
        return fo;
```
more clear and efficient